pkg/secrets/passphrase/manager_test.go

```diff
@@ -28,23 +28,15 @@ const (
 	brokenState = `{"salt":"fozI5u6B030=:v1:F+6ZduL:PGMFeIzwobWRKmEAzUdaQHqC5mMRIQ=="}`
 )
 
-func resetPassphraseTestEnvVars() func() {
-	clearCachedSecretsManagers()
-
-	oldPassphrase := os.Getenv("PULUMI_CONFIG_PASSPHRASE")
-	oldPassphraseFile := os.Getenv("PULUMI_CONFIG_PASSPHRASE_FILE")
-	return func() {
-		os.Setenv("PULUMI_CONFIG_PASSPHRASE", oldPassphrase)
-		os.Setenv("PULUMI_CONFIG_PASSPHRASE_FILE", oldPassphraseFile)
-	}
-}
-
 //nolint:paralleltest // mutates environment variables
 func TestPassphraseManagerIncorrectPassphraseReturnsErrorCrypter(t *testing.T) {
-	resetEnv := resetPassphraseTestEnvVars()
-	defer resetEnv()
+	clearCachedSecretsManagers()
 
-	os.Setenv("PULUMI_CONFIG_PASSPHRASE", "password123")
+	// There is no t.Unsetenv, so we set the environment variables to empty strings using t.Setenv and then unset them
+	// using os.Unsetenv. In doing do, the cleanup of t.Setenv takes care of resetting the environment variables when the
+	// test has completed.
+	t.Setenv("PULUMI_CONFIG_PASSPHRASE", "password123")
```

Comment:
Nit: In this case (and some others below) the comment is confusing because we're _not_ setting it to an empty string on this line. Should the comment be moved to under be under this line (and the similar cases below)?

Reply:
Will do :+1: